kernel_crawler/__init__.py

```diff
@@ -56,4 +61,12 @@ def crawl_kernels(distro, version, arch, to_driverkit):
             else:
                 ret[distname] = res
 
+    for distname, container in CONTAINER_DISTROS.items():
+        if distname == distro:
```

Comment:
Nice idea, changed as requested. 
So now with latest commit it will allow multiple images e.g. 
```
[jbrydon@jbrydon-dev-C8 kc_mr]$ python3 __init__.py crawl --distro redhat --image registry.redhat.io/rhel:registered --image registry.redhat.io/rhel7:registered --image redhat/ubi8:registered --out_fmt driverkit
```
OR
```
[jbrydon@jbrydon-dev-C8 kc_mr]$ python3 __init__.py crawl --distro * --image registry.redhat.io/rhel:registered --image registry.redhat.io/rhel7:registered --image redhat/ubi8:registered --out_fmt driverkit
```